src/fidesops/ops/models/privacy_request.py

```diff
@@ -505,27 +506,50 @@ def cache_manual_webhook_input(
             parsed_data.dict(),
         )
 
-    def get_manual_webhook_input(
+    def get_manual_webhook_input_strict(
```

Comment:
Updated this existing method to "strict mode" which used to just look for extra fields and/or no data cached, but now it also looks for missing fields.